docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.4'
```

Comment:
I would write in README.md or similar, how to test the e-mails

Reply:
I mean the `docker-compose -f DIR/docker-compose.mailhog.yaml up` command and the ports mapping information.